x-pack/platform/plugins/shared/aiops/public/cases/constants.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/**
+ * Titles for the cases toast messages
+ */
+export const TITLES = {
+  CHANGE_POINT_DETECTION: 'Change point detection',
```

Comment:
Should these be internationalized?

Reply:
`Change point chart` might be better here. `Change point chart added to case`

Reply:
Done in: [#7f408c6](https://github.com/elastic/kibana/pull/203813/commits/7f408c657c636217c0c2b20650d57b6dbfc54300)

Reply:
Sorry I'd forgotten that it's possible to multi-select charts to add to a case - can a plural condition be added for `Change point chart` / `Change point charts`?

Reply:
Added plural for `Change point charts`.
Also `Change point table` message when the table view is selected in: [#847d00e](https://github.com/elastic/kibana/pull/203813/commits/847d00e588bedf5aa4db8841aea3142ec7213c77)